src/app/contexts/RequestContext/getOriginContext.js

```diff
@@ -4,6 +4,10 @@ const getOriginContext = bbcOrigin => {
 
   if (bbcOrigin) {
     origin = bbcOrigin;
+  } else if (process && process.env && process.env.APP_ENV === 'local') {
```

Comment:
deepGet?

Reply:
This reads more clearly than `const appEnv = deepGet(['env', 'APP_ENV'], process);` so I'd rather not use it here.